ouroboros-consensus/bench/ChainSync-client-bench/Bench/Consensus/ChainSyncClient/Driver.hs

```diff
@@ -0,0 +1,128 @@
+{-# LANGUAGE LambdaCase #-}
+
+module Bench.Consensus.ChainSyncClient.Driver (mainWith) where
+
+import           Control.Monad (when)
+import qualified Data.Array.IO as UA
```

Comment:
Why `UA`?

Reply:
I had the same question, and saw later on that it's because of the unboxed arrays in the module (I think). Do we have any guidelines when it comes to naming imports?

Reply:
We don't really have many conventions. The only ones I can think of from the top of my head are:
- `Ledger.Core` is usually `Core` or `LC`
- `Consensus.Util.IOLike` as `IOLike`
- `Data.X.Strict` as `X` (with `X` being a container say `Map` or `Set`).
- `CBOR....` as `CBOR`
- `...AnchoredFragment` as `AF`
- `...AnchoredSeq` as `AS`
- `Data.ByteString` as `BS`
- `Data.ByteString.Lazy` as `BSL`
- `Data.Text` as `Text` or `T`
- `Test.StateMachine.*` as `QSM`
But we are not very consistent with this I think.